core-graphics/src/image.rs

```diff
@@ -163,3 +219,19 @@ extern {
     //fn CGImageGetAlphaInfo(image: ::sys::CGImageRef) -> CGImageAlphaInfo;
     //fn CGImageCreateCopyWithColorSpace(image: ::sys::CGImageRef, space: ::sys::CGColorSpaceRef) -> ::sys::CGImageRef
 }
+
+#[link(name = "ImageIO", kind = "framework")]
+extern "C" {
```

Comment:
Does this mean it should be in a subproject called image-io?

Reply:
Yes, it'd be better to make it a separate crate, so that users of the crate can control which frameworks they depend on.